pkg/reconciler/workload/placement/placement_controller.go

```diff
@@ -80,6 +80,17 @@ func NewController(
 		locationLister:  locationInformer.Lister(),
 		locationIndexer: locationInformer.Informer().GetIndexer(),
 
+		listLocations: func(clusterName logicalcluster.Name) ([]*schedulingv1alpha1.Location, error) {
+			locations, err := locationInformer.Lister().Cluster(clusterName).List(labels.Everything())
+			if errors.IsNotFound(err) {
```

Comment:
does this make sense? Can a list return NotFound?

Reply:
Technically yes if the resource does not exist